resultaten-versie-0.1.md

```diff
@@ -4,16 +4,15 @@
     Auteurs       : Anne Leemans/Fraukje Coopmans
```

Comment:
Ik ben echt super onder de indruk van deze resultaten, en dan bedoel ik eigenlijk niet eens inhoudelijk maar gewoon hoe duidelijk en netjes je deze hebt opgeschreven! Je hebt ook heel aardig mijn naam er boven gezet, maar dit is gewoon 99% jouw werk.